me.orszulak.keyring.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
looks like you don't even install this file.

Reply:
My bad, I'll fix that